src/__tests__/MainHomePage.test.js

```diff
@@ -0,0 +1,50 @@
+import {
+  render,
+  screen,
+  act,
+  waitFor,
+  fireEvent,
+} from "@testing-library/react";
+import { mswServer } from "./msw/msw-server";
+import App from "../App";
+
+beforeAll(() => mswServer.listen());
+afterEach(() => mswServer.resetHandlers());
+afterAll(() => mswServer.close());
+
+describe("Testing Main Component - Home Page", () => {
+  it("2.1. Featured Banners Slider is fetching and rendering data from the API ", async () => {
+    render(<App />);
+
+    await waitFor(() => { 
+      //Testing slider section
+      expect(screen.getByText("Slider Section")).toBeInTheDocument();
+      const image = screen.getByAltText("A GREAT STYLE - LIVING ROOM");
+      expect(image.src).toContain(
+        "https://images.prismic.io/wizeline-academy/305e2781-5f25-4c00-bef7-1041b49def37_banner-1-2.jpeg?auto=compress,format&rect=103,0,1226,600&w=1440&h=705"
+      );
+    });
+  });
```

Comment:
For tests is recommended to starts with  `it('should ....')`.
Review here when to use `getBy` vs `queryBy` vs `findBy` https://www.bam.tech/en/article/getby-findby-queryby-whats-the-difference https://sheelahb.com/blog/react-testing-library-tips-and-tricks/

```suggestion
  // 2.1. Featured Banners Slider is fetching and rendering data from the API 
  it("should fetch and render the Featured Banners from the API", async () => {
    render(<App />);
    
    // Notes for Daniel: you should wait until the loading finished
    
    const sliderContainer = await screen.findByTestId('slider-section') // Add `data-testid='slider-section'` to the featured banners div // Use await with `.find` for wait to elements appears in the screen, this means the loading finished
    
    const sliderSection = within(sliderContainer).getByText("Slider Section"); // Div element 
    expect(sliderSection).toBeInTheDocument(); // use `within` function to search only in the featured banners section instead of the whole screen // Here is not needed the `await .find..` because above you waited for the slider section appears in the document
    
    const image =  within(sliderContainer).getByAltText("A GREAT STYLE - LIVING ROOM");
      expect(image.src).toContain(
        "https://images.prismic.io/wizeline-academy/305e2781-5f25-4c00-bef7-1041b49def37_banner-1-2.jpeg?auto=compress,format&rect=103,0,1226,600&w=1440&h=705"


    });
  });
```

Reply:
done